src/Uno.UWP/Storage/Helpers/StorageFileHelper.cs

```diff
@@ -0,0 +1,13 @@
+﻿#nullable enable
+using System.Threading.Tasks;
+namespace Windows.Storage.Helpers;
```

Comment:
New line for readability

```suggestion

namespace Windows.Storage.Helpers;
```